src/sentry/static/sentry/app/views/settings/organizationTeams/teamSettings/index.jsx

```diff
@@ -52,6 +52,7 @@ export default class TeamSettings extends AsyncView {
       this.props.router.push(
         `/settings/${this.props.params.orgId}/teams/${model.getValue(id)}/settings/`
       );
+      this.forceUpdate();
```

Comment:
What does `forceUpdate` here? Wouldn't `setState` also cause a re-render?

Reply:
`setState` doesn't always immediately update the component and it could batch/defer the update until later -- which I think would continue this error. 
whereas `forceUpdate` skips the `shouldComponentUpdate`.  

I'm not 100% set on it though. What do you think?

Reply:
I think we should add a comment explaining this.

Reply:
We didn't end up doing this at all